examples/md-flexible/src/ParallelVtkWriter.h

```diff
@@ -76,17 +80,46 @@ class ParallelVtkWriter {
    */
   int _maximumNumberOfDigitsInIteration;
 
+  /**
+   * Writes the current state of particles into vtk files.
+   * @param currentIteration: The simulations current iteration.
+   * @param autoPasContainer The AutoPas container whose owned particles will be logged.
+   */
+  void recordParticleStates(const int &currentIteration, const autopas::AutoPas<ParticleType> &autoPasContainer);
+
+  /**
+   * Writes the current domain subdivision into vtk files.
+   * @param currentIteration: The simulations current iteration.
+   * @param decomposition: The simulations domain decomposition.
+   */
+  void recordDomainSubdivision(const int &currentIteration, const autopas::Configuration &autoPasConfiguration,
+                               const RegularGridDecomposition &decomposition);
+
+  /**
+   * Calculates the whole extent of the decompositions local domain.
+   * The whole extent defines the space this local domain is occupying in the global domain.
+   * The layout of the returned array is [ xmin, xmax, ymin, ymax, zmin, zmax ].
+   * @param decomposition: The simulations domain decomposition.
+   * @return the whole extent of the local domain.
+   */
+  std::array<int, 6> calculateWholeExtent(const RegularGridDecomposition &domainDecomposition);
+
   /**
    * Tries to create a folder for the current writer session and stores it in _sessionFolderPath.
    */
   void tryCreateSessionAndDataFolders(const std::string &name, const std::string location);
 
   /**
-   * Creates the .pvtu file required to load data from multiple ranks into paraview.
+   * Creates the .pvtu file required to load unstrictured grid data from multiple ranks into paraview.
```

Comment:
```suggestion
   * Creates the .pvtu file required to load unstructured grid data from multiple ranks into ParaView.
```

So is the grid structured or unstructured? The comment in line 119 says structured...